package.json

```diff
@@ -6,39 +6,40 @@
   "dependencies": {
     "@chakra-ui/next-js": "^2.1.5",
     "@chakra-ui/react": "^2.8.1",
-    "@cosmjs/cosmwasm-stargate": "^0.31.3",
-    "@cosmjs/stargate": "^0.31.3",
-    "@cosmos-kit/core": "^2.7.2",
-    "@cosmos-kit/cosmostation": "^2.4.4",
-    "@cosmos-kit/keplr": "^2.4.4",
-    "@cosmos-kit/leap": "^2.4.3",
-    "@cosmos-kit/react": "^2.5.3",
+    "@cosmjs/cosmwasm-stargate": "^0.32.3",
+    "@cosmjs/stargate": "^0.32.3",
```

Comment:
If you upgrade this, you'll need to make sure the package patch that was applied for the injective blockchain still works. See the `injective` branch